package.json

```diff
@@ -64,7 +64,12 @@
     "typescript": "^5.1.6"
   },
   "workspaces": [
-    "packages/*",
+    "packages/cloud-core",
```

Comment:
The glob pattern should be fine, any reason why this was changed? Manually doing this become hard to manage as more packages are added

Reply:
Ok, if it becomes troublesome in the future we can revisit & enhance the build process.